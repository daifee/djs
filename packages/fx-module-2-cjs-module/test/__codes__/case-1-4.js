
fxDefine('case-4', {
  age: 18,
  name: 'lili'
});
