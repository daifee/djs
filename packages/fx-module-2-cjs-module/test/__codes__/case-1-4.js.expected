"use strict";

module.exports = {
  age: 18,
  name: 'lili'
};